quipucords/scanner/job.py

```diff
@@ -37,6 +37,29 @@ def get_task_runner_class(scan_task: ScanTask):
     raise NotImplementedError
 
 
+def get_task_runners_for_job(
+    scan_job: ScanJob,
+) -> tuple[list[ScanTaskRunner], FingerprintTaskRunner | None]:
+    """Get ScanTaskRunners for all the tasks that need to run for this job."""
+    incomplete_scan_tasks = scan_job.tasks.filter(
```

Comment:
dummy question. why using Q here? couldnt it be:
```
    incomplete_scan_tasks = scan_job.tasks.filter(
        status__in=[ScanTask.RUNNING, ScanTask.PENDING]
    ).order_by("sequence_number")
```

Reply:
Updated and pushed! Let me know if you spotted other filters like this, and I'm happy to update.